### ISSUE TITLE
Accessibility "Support-Button" (Voice-Reader)

### ISSUE THREAD

**Issue by jostertag** [liqd/a4-meinberlin#4664]
**URL:** https://meinberlin-dev.liqd.net/projekte/module/burgerinnenhaushalt-3-phasen-21/?mode=list
**user:** any
**expected behaviour:** For accessibility the voice reader should tell the users how many people support a proposal in the detailed proposal view, the list view and the map view. In the proposal view the voice reader should tell the users, if they already clicked the support-button or not.
**behaviour:** The voice reader (any) doesn't read out loud the number of supporters in the proposal tiles nor in the detailed proposal view. In the detailed proposal view, the voice reader just reads out loud "click to support", but not how many people already support the proposal. In the detailed proposal view, the voice reader cannot tell the user whether the user has already clicked the support-button or not.
**important screensize:** any
**device & browser:** any
**Comment/Question:** It wasn't explicitly in a story and I don't know if it's important right now, just realized that this is a problem for accessibility.

Screenshot?
![Bildschirmfoto 2022-11-08 um 11 41 30](https://user-images.githubusercontent.com/113608720/200543881-28cae1fe-fa54-4715-a674-91400b4cd910.png)

![Bildschirmfoto 2022-11-08 um 11 42 11](https://user-images.githubusercontent.com/113608720/200544100-e17e68d5-5779-4059-8af6-72688ae34a93.png)

![Bildschirmfoto 2022-11-08 um 11 42 30](https://user-images.githubusercontent.com/113608720/200544115-47101429-c7e5-4be5-a245-087e4c654a17.png)

**Comment by khamui** [comment]
is there already a translation/wording which we could use? if not, what would be a the text to be read out @jostertag @CarolingerSeilchenspringer ?

**Comment by jostertag** [comment]
@khamui I'll look into it in detail next week to find a suitable wording. Is it possible to have a different wording depending on whether the user already clicked on the button or not? And is it possible that the wording includes the actual number of people who support the idea?

**Comment by khamui** [comment]
@jostertag we checked in our code and we can include the number in the text. But showing if the user clicked the button or not needs a little story, as this logic has to be added.

Does that help?

**Comment by jostertag** [comment]
thank you @khamui for the info. That helps.

 @CarolingerSeilchenspringer I would suggest this wording then:
"1 person supports this proposal. Click to support."
"1 Person unterstützt diesen Vorschlag. Zum Unterstützen klicken."
"% persons support this proposal. Click to support."
"% Personen unterstützen diesen Vorschlag. Zum Unterstützen klicken."

**Comment by khamui** [comment]
@jostertag @CarolingerSeilchenspringer  while working on that, I realised the following, which led to the decision to remove the second sentence <kbs>Click to support.<kbs>.

- In the Map pin: user cannot vote as it isn't a button
- In the list: user cannot vote as it isn't a button
- On the detail page: here we could actually distinguish if the user clicked or not. Sorry for the contradiction, as I said before it is not possible 😬.

In the end for all three cases the second sentence would not make much sense. I hope that is okay for now.

**Comment by jostertag** [comment]
@khamui yes with the second sentence I was especially refering to the detail page. If you cannot distinguish between map, list and detail page, then just the first sentence makes sence. I think we can leave that for now and keep it in mind for the future or what do you think @CarolingerSeilchenspringer ?

**Comment by jostertag** [comment]
@khamui Okay now I get it. 

My suggestion for the wording on the detail page would be:
**if user didn't click on it:**
"1 person supports this proposal. Click to support."
"1 Person unterstützt diesen Vorschlag. Zum Unterstützen klicken."
"% persons support this proposal. Click to support."
"% Personen unterstützen diesen Vorschlag. Zum Unterstützen klicken."

**if user already clicked on it:**
"1 person supports this proposal. You support this proposal. Click to unsupport."
"1 Person unterstützt diesen Vorschlag. Sie unterstützen diesen Vorschlag. Zum Entziehen der Unterstützung klicken."
"% persons support this proposal. You support this proposal. Click to unsupport."
"% Personen unterstützen diesen Vorschlag. Zum Entziehen der Unterstützung klicken."

**as discussed for the map pin and list view just this wording:** 
"1 person supports this proposal." "% persons support this proposal."
